packages/expo/src/winter/FormData.ts

```diff
@@ -70,7 +70,16 @@ function normalizeArgs(
   blobFilename: string | undefined
 ): [string, File | string] {
   if (value instanceof Blob) {
-    value = { type: value.type, name: blobFilename || 'blob', blob: value };
+    // @ts-expect-error: `Blob.data.blobId` is react-native's proprietary property.
+    if (value.data?.blobId != null) {
+      // For react-native created Blob objects,
+      // we need to keep its original form as-is without breaking functionality.
+      // However, we need to pass `name` for our file name handling.
+      // @ts-expect-error: Mutating the Blob object to add the `name` property.
+      value.name = blobFilename || 'blob';
```

Comment:
Should this be `??`?

Reply:
was just thinking to prevent people passing empty string like `''`. it turns out that undici allows empty string, then that's fine: https://github.com/nodejs/undici/blob/6bb527e9d768c5ca5b98765c28cf403ad8b250fb/lib/web/fetch/formdata.js#L246

i'm revising it as `??`. thanks!